app/src/main/java/com/celzero/bravedns/util/Constants.kt

```diff
@@ -122,13 +124,39 @@ class Constants {
         const val PORT_VAL_UNKNOWN = "unknown"
 
         const val NOTIFICATION_ACTION = "NOTIFICATION_VALUE"
-        const val VPN_NOTIFICATION_ACTION = "STOP_RETHINKDNS"
+        const val STOP_VPN_NOTIFICATION_ACTION = "RETHINK_STOP"
+        const val DNS_VPN_NOTIFICATION_ACTION = "RETHINK_DNSONLY" // battery-saver dns-only
+        const val DNS_FIREWALL_VPN_NOTIFICATION_ACTION = "RETHINK_FULLMODE" // default dns+firewall
+        const val RULES_FAILURE_NOTIFICATION_ACTION = "RETHINK_RULES_RELOAD" //
```

Comment:
stray comment `//`? run the code-formatter. pls do not send code reviews without running it.

Reply:
Added comment.